xbmc/cores/dvdplayer/DVDCodecs/Video/AMLCodec.cpp

```diff
@@ -1427,6 +1371,10 @@ CAMLCodec::CAMLCodec() : CThread("CAMLCodec")
   am_private = new am_private_t;
   memset(am_private, 0, sizeof(am_private_t));
   m_dll = new DllLibAmCodec;
+  if(!(m_dll->CanLoad() && m_dll->Load())) {
```

Comment:
CanLoad doesn't work on Android. Does it on Linux?

Also please follow formatting guidelines:
- 2 spaces indent (for info, haven't seen an issue)
- Opening bracket on own line